system/threads.cpp

```diff
@@ -197,6 +197,11 @@ long __syscall_set_tid_address(int *tidptr)
 	return tid;
 }
 
+long __syscall_membarrier(int cmd, unsigned int flags)
+{
+	return ENOSYS;
```

Comment:
I doubt that anybody checks the return value for this. but check what is the success return value (I guess '0') and return that

Reply:
It now returns 0 to avoid fallback code in musl.